ci-operator/step-registry/ipi/conf/osimage/nutanix/ipi-conf-osimage-nutanix-commands.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+if [[ -z ${CLUSTER_OS_IMAGE} ]]; then
+    echo "Get default clusterOSImage"
+    CLUSTER_OS_IMAGE=$(curl -s 'https://raw.githubusercontent.com/openshift/installer/master/data/data/coreos/rhcos.json' | jq -r '.architectures.x86_64.artifacts.nutanix.formats.qcow2.disk.location')
```

Comment:
I think we should use `openshift-install coreos print-stream-json` to get the default os image instead of curling in the installer repo's master branch.